docs/reference/expectations/result_format.md

```diff
@@ -4,66 +4,62 @@ title: Result format
 
 
 The `result_format` parameter may be either a string or a dictionary which specifies the fields to return in `result`.
-  * For string usage, see `result_format` values.
+  * For string usage, see [`result_format` values and fields](#result_format-values-and-fields).
   * For dictionary usage, `result_format` which may include the following keys:
     * `result_format`: Sets the fields to return in result.
+    * `unexpected_index_column_names`: Defines the primary key (PK) columns used to represent unexpected results.
```

Comment:
How about something like the following? Just to be more general that a column does not have to be the primary key column and explicit that multiple columns can be used.
```suggestion
    * `unexpected_index_column_names`: Defines columns that can be used to identify unexpected results, for example primary key (PK) column(s) or other columns with unique identifiers. Supports multiple column names as a list.
```

Reply:
Love this change. thank you :)